UnbxdSearch_IO.js

```diff
@@ -1073,7 +1073,7 @@ var unbxdSearchInit = function (jQuery, Handlebars) {
             return this;
         },
         addFilter: function (field, value) {
-            if (this.options.facetMultilevel && field === (this.options.mappedFields.categoryField || 'categoryPath')) {
+            if (this.options.facetMultilevel && field === (this.options.mappedFields.categoryPath || 'categoryPath')) {
```

Comment:
why we need this change.

Reply:
We have an API for dimension map fields. Which returns object 
`{categoryPath: "ProductCategory"
description: "Description"
imageUrl: "ImageURL"
price: null
productUrl: null
title: "title"}`
So there is a change in the key from `categoryField`->`categoryPath`.

Reply:
@sanjithkumar017 
Option 1:
In that case we need to make this change in the sdk also, from mappedfields.categoryfield to mappedfields.categorypath. 
The name was supposed to be generic, hence the terminology categoryfield.
Option 2:
We can make the response change in OnboardingMW.js, and change categoryPath to categoryField, as that is correct terminology.

Reply:
Yes, It's better to change the response.